scripts/helm_gke.sh

```diff
@@ -7,35 +7,34 @@
 # NOTE! If you are using gsuite, you should also set `authenticate.idp.serviceAccount`, see docs !
 
 echo "=> [GCE] creating cluster"
-gcloud container clusters create pomerium --region us-west2
+gcloud container clusters create pomerium --region us-west2 --num-nodes 1
 
 echo "=> [GCE] get cluster credentials so we can use kubctl locally"
 gcloud container clusters get-credentials pomerium --region us-west2
 
 echo "=> add pomerium's helm repo"
 helm repo add pomerium https://helm.pomerium.io
 
+echo "=> add bitnami's helm repo"
```

Comment:
Our demo policy targeted a service that wasn't present.  Added nginx chart serving a hello page instead.